From 0caeff6831dde87caf070dfc13399401fb9908fe Mon Sep 17 00:00:00 2001
From: James Futhey <kidGodzilla@users.noreply.github.com>
Date: Mon, 21 Apr 2025 11:31:18 +0800
Subject: [PATCH] rm support link

---
 src/screens/Signup/index.tsx | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/src/screens/Signup/index.tsx b/src/screens/Signup/index.tsx
index c98040010..e12b1bfa4 100644
--- a/src/screens/Signup/index.tsx
+++ b/src/screens/Signup/index.tsx
@@ -5,7 +5,7 @@ import {AppBskyGraphStarterpack} from '@atproto/api'
 import {msg, Trans} from '@lingui/macro'
 import {useLingui} from '@lingui/react'
 
-import {FEEDBACK_FORM_URL} from '#/lib/constants'
+// import {FEEDBACK_FORM_URL} from '#/lib/constants'
 import {useServiceQuery} from '#/state/queries/service'
 import {useStarterPackQuery} from '#/state/queries/starter-packs'
 import {useActiveStarterPack} from '#/state/shell/starter-pack'
@@ -24,7 +24,7 @@ import {atoms as a, useBreakpoints, useTheme} from '#/alf'
 import {AppLanguageDropdown} from '#/components/AppLanguageDropdown'
 import {Divider} from '#/components/Divider'
 import {LinearGradientBackground} from '#/components/LinearGradientBackground'
-import {InlineLinkText} from '#/components/Link'
+// import {InlineLinkText} from '#/components/Link'
 import {Text} from '#/components/Typography'
 import * as bsky from '#/types/bsky'
 
@@ -186,16 +186,16 @@ export function Signup({onPressBack}: {onPressBack: () => void}) {
             <View
               style={[a.w_full, a.py_lg, a.flex_row, a.gap_lg, a.align_center]}>
               <AppLanguageDropdown />
-              <Text
-                style={[t.atoms.text_contrast_medium, !gtMobile && a.text_md]}>
-                <Trans>Having trouble?</Trans>{' '}
-                <InlineLinkText
-                  label={_(msg`Contact support`)}
-                  to={FEEDBACK_FORM_URL({email: state.email})}
-                  style={[!gtMobile && a.text_md]}>
-                  <Trans>Contact support</Trans>
-                </InlineLinkText>
-              </Text>
+              {/*<Text*/}
+              {/*  style={[t.atoms.text_contrast_medium, !gtMobile && a.text_md]}>*/}
+              {/*  <Trans>Having trouble?</Trans>{' '}*/}
+              {/*  <InlineLinkText*/}
+              {/*    label={_(msg`Contact support`)}*/}
+              {/*    to={FEEDBACK_FORM_URL({email: state.email})}*/}
+              {/*    style={[!gtMobile && a.text_md]}>*/}
+              {/*    <Trans>Contact support</Trans>*/}
+              {/*  </InlineLinkText>*/}
+              {/*</Text>*/}
             </View>
           </View>
         </View>
-- 
2.39.5 (Apple Git-154)

